.idea/misc.xml

```diff
@@ -9,7 +9,7 @@
     <resource url="http://schemas.android.com/apk/res-auto" location="$PROJECT_DIR$" />
     <resource url="http://schemas.android.com/apk/res/android" location="$PROJECT_DIR$" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Are we using JDK 1.8 ?

Reply:
I am on JDK1.8 don't know why this would have changed 🤔

Reply:
Mine is on JDK_1_8 too, compile options and kotlin options in common.gradle both are V_1_8 as well.

Reply:
Yeh like I said I don't know why this would have changed- it's autogenerated but I haven't changed anything that isn't ios. Anyway, I changed it back to what it was